source/module_base/spherical_bessel_transformer.cpp

```diff
@@ -41,10 +164,9 @@ long long int SphericalBesselTransformer::spherical_bessel_sincos_polycoef(const
      *      c(1,1) = -1     c(1,1) = 0
      *                                                              */
     assert(l >= 0 && n >= 0);
-    assert(l <= 17 && "SphericalBesselTransformer::spherical_bessel_sincos_polycoef: \
-                some coefficients exceed LLONG_MAX (2^63-1) for l >= 18");
+    assert(l <= 17 && "Impl::polycoef: some coefficients exceed LLONG_MAX (2^63-1) for l >= 18");
```

Comment:
Please bracket "assert" with HONG __DEBUG  for high performance.

Reply:
> Please bracket "assert" with HONG __DEBUG for high performance.

I notice that, apart from those in module_nao, there are still numerous "assert" across the code. If we want to skip all assert in release build, shall we consider using the standard macro "NDEBUG"?
https://en.cppreference.com/w/cpp/error/assert